src/factories/calorimetry/CalorimeterClusterRecoCoG_factoryT.h

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023, Wouter Deconinck
+// Subject to the terms in the LICENSE file found in the top-level directory.
+//
+
+#pragma once
+
+#include <algorithms/calorimetry/CalorimeterClusterRecoCoG.h>
+#include <services/geometry/dd4hep/JDD4hep_service.h>
+#include <extensions/jana/JChainMultifactoryT.h>
+#include <extensions/spdlog/SpdlogMixin.h>
+
+
+namespace eicrecon {
+
+template<class T>
+class CalorimeterClusterRecoCoG_factoryT :
+    public JChainMultifactoryT<CalorimeterClusterRecoCoGConfig>,
+    public SpdlogMixin<T> {
+
+  public:
+    using SpdlogMixin<T>::logger;
+
+    explicit CalorimeterClusterRecoCoG_factoryT(
+        std::string tag,
+        const std::vector<std::string>& input_tags,
+        const std::vector<std::string>& output_tags,
+        CalorimeterClusterRecoCoGConfig cfg)
+    : JChainMultifactoryT<CalorimeterClusterRecoCoGConfig>(std::move(tag), input_tags, output_tags, cfg) {
+
+      DeclarePodioOutput<edm4eic::Cluster>(GetOutputTags()[0]);
+      DeclarePodioOutput<edm4eic::MCRecoClusterParticleAssociation>(GetOutputTags()[1]);
+
+    }
+
+    //------------------------------------------
+    // Init
+    void Init() override {
+
+        auto app = GetApplication();
+
+        // This prefix will be used for parameters
+        std::string plugin_name  = eicrecon::str::ReplaceAll(GetPluginName(), ".so", "");
+        std::string param_prefix = plugin_name + ":" + GetTag();
```

Comment:
This would set the prefix, I think.
```suggestion
        std::string param_prefix = plugin_name + ":" + GetTag();
        JChainMultifactoryT<CalorimeterClusterRecoCoGConfig>::SetPrefix(param_prefix);
```

Reply:
Nevermind, probably irrelevant.